### PR TITLE
added dispatch_action_config to PlainTextInput

### PR DESCRIPTION
**Pull request by stevengill**
###  Summary

Fixes issue #1108 

### Requirements (place an `x` in each `[ ]`)

* [x] I've read and understood the [Contributing Guidelines](https://github.com/slackapi/node-slack-sdk/blob/master/.github/contributing.md) and have done my best effort to follow them.
* [x] I've read and agree to the [Code of Conduct](https://slackhq.github.io/code-of-conduct).

### CODE REVIEW

File: packages/types/src/index.ts [hunk 1]
```diff
@@ -241,6 +241,11 @@ export interface PlainTextInput extends Action {
   multiline?: boolean;
   min_length?: number;
   max_length?: number;
+  dispatch_action_config?: DispatchActionConfig;
+}
+
+export interface DispatchActionConfig {
+  trigger_actions_on?: ('on_enter_pressed' | 'on_character_entered')[];
```
Comment (stevengill) [inline packages/types/src/index.ts:248]: I did this instead of `string[]` as it is more accurate.

File: packages/types/src/index.ts [hunk 2]
```diff
@@ -241,6 +241,11 @@ export interface PlainTextInput extends Action {
   multiline?: boolean;
   min_length?: number;
   max_length?: number;
+  dispatch_action_config?: DispatchActionConfig;
```
Comment (stevengill) [inline packages/types/src/index.ts:244]: https://api.slack.com/reference/block-kit/block-elements#input is missing if `dispatch_action_config` is required or not. I'm assuming it isn't required.

Comment (mwbrooks) [reply packages/types/src/index.ts:244]: Good catch. We should probably report this to the api.slack.com team as well

**Review by mwbrooks — APPROVED** [review]

File: packages/types/src/index.ts (hunk 2 above)
Comment (stevengill) [reply packages/types/src/index.ts:244]: Yup! I reported it to them!

**Review by seratch — APPROVED** [review]

File: packages/types/src/index.ts (hunk 2 above)
Comment (seratch) [reply packages/types/src/index.ts:244]: >I'm assuming it isn't required.

This field is surely optional.

**Pull request merged**